### ISSUE TITLE
Add new branch of build

### ISSUE THREAD

**Issue by mcc-devel** [zero9178/GNU-Toolchain-for-Windows#1]
Gcc now actually supports C++20 modules now, and you can get the source from [gitee](https://gitee.com/mirrors/gcc) using the following command:
```shell
git clone https://gitee.com/mirrors/gcc.git -b devel/c++-modules --depth=1
```
That way, it supports the following syntax:
```c++
import std.core;
```

**Comment by zero9178** [comment]
The way GCC builds are able to live in parallel this would cause builds of GCC with modules to override GCC-11 builds which isn't ideal since the modules branch does not rebase that regularly with the master branch. 

As soon as modules are merged into the master branch, which should hopefully be soon, I'll be updating the GCC-11 and master branch as fast as possible.

**Comment by mcc-devel** [comment]
Thanks for your response, I hope gcc will merge soon.
